_posts/2016-03-28-artsy-engineering-organization-stack.markdown

```diff
@@ -0,0 +1,55 @@
+
+Each product team consists of one or more _Business People_, a _Product Owner_ (often a _Product Manager_), a _Designer_, an _Engineering Lead_ and several _Engineers_.
+
+Our product teams map directly into our businesses and inherit the same KPIs.
+
+**Partner Success**: The Partner Success team focuses on products that serve the Listings business. These include our Content Management system and [Artsy Folio](http://folio.artsy.net), an iPad app used by our partners.
+
+**Auctions**: The Auctions product team builds all auction systems across web and mobile.
+
+**Collector GMV**: GMV stands for _Gross Merchandising Volume_, a measure of artworks traded on the platform. We have two collector product teams, web and mobile, focused on various customer experiences across all our web and mobile properties.
+
+**Publishing**: The publishing team works on the platform used to produce all our online editorial content.
+
+## Practices
+
+While there're many advantages of product teams operating independently, it's also very easy to create vertical inefficiencies. Multiple teams could choose to achieve similar outcomes in many different ways, creating a lot of redundant work and making it impossible for team members to move between organizations. To address this problem we've created a separate _Platform Engineering Team_, responsible for systems and shared infrastructure that help product teams work smarter, scale our operations, and much more. Our _Design_ team is similar in nature to Platform Engineering, creating a consistent visual language and user experience across all products.
+
+Finally, we also have _Practices_, specifically _Web_ and _Mobile_, where a handful of very senior Engineers work independently and float between Product teams to build features while evangelizing best practices of making software or our [commitment to open-source](http://artsy.github.io/open-source).
+
+## Reporting Structure
+
+We believe that in order to help you grow in your individual contributor role, your manager must understand your work. Engineers report to Engineering Leads in product teams. Engineering and Engineering Practice Leads report to the Head of Engineering. Product Managers report to the Head of Product. Designers report to the Design Lead, who reports to the Head of Product. It's important to note that, while preserving top-down authority and responsibility, we try to practice bottom-up leadership - the job of team leads is to help individual contributors do their best work.
```

Comment:
> We believe that in order to help you grow in your individual contributor role, your manager must understand your work.

This is the first sentence to address the reader directly, and the only one to address them as a hypothetical Artsy team member. It feels out of place by itself, maybe consider spreading more throughout the rest, or rephrasing this sentence?